docs/standards/news.rst

```diff
@@ -8,19 +8,19 @@ Release Notes
 ==============
 
 
-Version 1.2.?: current
+Version 1.?.?: current
 --------------------------------------------------------------------------------
 
++ Added ontobee as default source for ``ontologies``
```

Comment:
We shifted from Ontobee to [EBI OLS](https://www.ebi.ac.uk/ols/) briefly before merging #296 (4d3c45c). Background were repeated connectivity issues with Ontobee and the additional cross-referencing service by OLS.

Reply:
didn't spot that before, I corrected it now.